src/main/java/org/folio/bulkops/domain/bean/Item.java

```diff
@@ -1,374 +1,313 @@
 package org.folio.bulkops.domain.bean;
 
-import static java.lang.Boolean.FALSE;
-import static java.util.Collections.emptyList;
-import static java.util.Objects.isNull;
-import static org.apache.commons.lang3.StringUtils.EMPTY;
-
-import java.util.List;
-
-import org.folio.bulkops.domain.converter.BooleanConverter;
-import org.folio.bulkops.domain.converter.BoundWithTitlesConverter;
-import org.folio.bulkops.domain.converter.CallNumberTypeConverter;
-import org.folio.bulkops.domain.converter.CirculationNoteListConverter;
-import org.folio.bulkops.domain.converter.ContributorListConverter;
-import org.folio.bulkops.domain.converter.DamagedStatusConverter;
-import org.folio.bulkops.domain.converter.EffectiveCallNumberComponentsConverter;
-import org.folio.bulkops.domain.converter.ElectronicAccessListConverter;
-import org.folio.bulkops.domain.converter.IntegerConverter;
-import org.folio.bulkops.domain.converter.ItemLocationConverter;
-import org.folio.bulkops.domain.converter.ItemNoteListConverter;
-import org.folio.bulkops.domain.converter.ItemStatisticalCodeListConverter;
-import org.folio.bulkops.domain.converter.ItemStatusConverter;
-import org.folio.bulkops.domain.converter.LastCheckInConverter;
-import org.folio.bulkops.domain.converter.LoanTypeConverter;
-import org.folio.bulkops.domain.converter.MaterialTypeConverter;
-import org.folio.bulkops.domain.converter.ServicePointConverter;
-import org.folio.bulkops.domain.converter.StringConverter;
-import org.folio.bulkops.domain.converter.StringListConverter;
-import org.folio.bulkops.domain.converter.TagsConverter;
-import org.folio.bulkops.domain.dto.IdentifierType;
-
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonTypeName;
 import com.opencsv.bean.CsvCustomBindByName;
 import com.opencsv.bean.CsvCustomBindByPosition;
-
 import jakarta.validation.Valid;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 import lombok.With;
+import org.folio.bulkops.domain.converter.*;
```

Comment:
@AntonAntonich please use import for all needed classes.

Reply:
done